src/css/component-styles/AppContainer.scss

```diff
@@ -19,16 +19,18 @@
   height: 100px;
 }
 
-.feedbackContainer {
-  z-index: 999;
-  color: #000;
-  position: fixed;
+.feedback-container {
+  background-color: #e6e6e6;
   bottom: 0px;
+  display: flex;
+  flex-direction: column;
```

Comment:
Changing feedback-container to flex messed up the positioning of the message. I attached the picture of the extra space highlighted in red.
![edit](https://user-images.githubusercontent.com/33295600/61974242-0be45c00-afb4-11e9-97e6-a0016d9c6d9d.png)

Reply:
fixed